atlp-rwanda/octopus-bn-frontend
# 170947380 - Users Should Receive  Reset Password Link

#### What does this PR do?
It allows a user to receive a link in their email address to reset their password.

#### Description of Task to be completed?

- send a password reset  link to the user's email address

#### How should this be manually tested?
-  Run this `git clone https://github.com/Stackup-Rwanda/octopus-bn-frontend.git` in your terminal if you don't have the project on your computer
- If you have it, run `git fetch && git checkout ft-reset-password-170947380` instead
- Run `npm run dev` command 
- In your browser go to `localhost:your_port/forgot-password`
- Enter an existing and accessible email in the field
- Go that same email address and you'll see the reset password email.

#### Screenshots (if appropriate)
 ##### Reset Password  Request Page

<img width="1438" alt="Screen Shot 2020-05-06 at 20 58 23" src="https://user-images.githubusercontent.com/25052255/81217353-c3df7480-8fdc-11ea-91ca-2be3101a25be.png">

#### What are the relevant pivotal tracker stories?

[Maintains  #170947380](https://www.pivotaltracker.com/story/show/170947380)
